sim/src/mechanics/car.rs

```diff
@@ -133,6 +133,33 @@ impl Car {
         };
 
         let body = match self.state {
+            CarState::ChangingLanes {
+                from,
+                to,
+                ref lc_time,
+                ..
+            } => {
+                let percent_time = 1.0 - lc_time.percent(now);
+                // TODO Can probably simplify this! Lifted from the parking case
+                let r = map.get_parent(from);
+                // The car's body is already at 'to', so shift back
+                let mut diff = (r.offset(to) as isize) - (r.offset(from) as isize);
+                if map.get_l(from).dir == Direction::Fwd {
+                    diff *= -1;
+                }
+                // TODO Careful with this width math
```

Comment:
If the source/target lanes have a different width, I think we have to do something different here. Need to first force a repro